src/core/hle/service/set/set.cpp

```diff
@@ -120,6 +159,35 @@ void SET::GetRegionCode(Kernel::HLERequestContext& ctx) {
     rb.Push(Settings::values.region_index);
 }
 
+void GetKeyCodeMapImpl(Kernel::HLERequestContext& ctx) {
+    const auto language_code = available_language_codes[Settings::values.language_index];
+    const auto key_code =
+        std::find_if(language_to_layout.begin(), language_to_layout.end(),
+                     [&](const auto& element) { return element.first == language_code; });
```

Comment:
```suggestion
                     [=](const auto& element) { return element.first == language_code; });
```

Given all that's being copied is a single u64